darts/tests/test_RNN.py

```diff
@@ -59,6 +59,13 @@ def test_fit(self):
         pred4 = model3.predict(n=1)
         self.assertEqual(len(pred4), 1)
 
+        # test validation series input
+        model3.fit(self.series[:60], val_series=self.series[60:])
+        pred4 = model3.predict(n=6)
+        self.assertEqual(len(pred4), 6)
```

Comment:
I believe test should be short and contain as little assertion as possible (usually 1) this allows to quickly see directly from the test runner what is failing and act accordingly. We haven't been following this practice in the past but it seems that this test could be split up and the method name could be more explicit, what do you think ?

Reply:
That's probably a good idea, but I think this should be done in a separate PR, as it would require a complete refactoring of the current test cases.